app/controllers/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UsersController < ApplicationController
+  skip_before_action :authenticate_user!
+  def show
+    @company = current_company
+    @users = @company.users.find_by_sequence_num!(params[:id])
+    flash.now[:notif] = "Welcome to Specific User Page of (#{@company.name})"
+  end
```

Comment:
Our convention is to include respond_to block even when you are only handling html format

Reply:
done

Reply:
Do it for index as well

Reply:
done